Makefile

```diff
@@ -41,24 +41,24 @@ validate: setup
 # Run tests
 .PHONY: test
 test: $(KUBEBUILDER)
-	go test -race -v -coverprofile cover.out ./...
+	go test -ldflags="-X github.com/cybozu-go/moco/test_utils.MySQLVersion=$(MYSQL_VERSION)" -race -v -coverprofile cover.out ./...
```

Comment:
I think you can use environment variable instead of embedding variable.
Which do you think is better?